### PR TITLE
improvement: Optimised building process of design system

### PR DESCRIPTION
**Pull request by daylesalmon**

### CODE REVIEW

File: src/www/pages/components/accordion/index.njk.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    js: true,
+    iframe: false
+  })
+}}
+
+{% markdown %}
+
+## Custom
+
+In the example below, you can see that we have cusomised the summary section of the accordion so
```
Comment (catiarodriguescosta) [inline src/www/pages/components/accordion/index.njk.md:111]:
```suggestion
In the example below, you can see that we have customised the summary section of the accordion so
```

**Review by catiarodriguescosta — APPROVED** [review]
Looks great, even indentation! Excellent work! 👍 
Just found minor typos inside comments - did create some suggestions to fix them.

**Review by gldgrnt — APPROVED** [review]
Everything looks great 👍🏾 Just thinking, when this approved / checked by everyone, this should probably be merged into release via **merge commit** so that all of the feat/fix commits show up in the changelog.

**Review by catiarodriguescosta — APPROVED** [review]
👍

**Review by gldgrnt — APPROVED** [review]
👍🏾

**Pull request merged**